common/src/main/java/io/netty5/util/concurrent/GlobalEventExecutor.java

```diff
@@ -91,7 +90,7 @@ this, newPromise(), Executors.callable(() -> {
                 deadlineNanos(ticker().nanoTime(), SCHEDULE_QUIET_PERIOD_INTERVAL),
                 -SCHEDULE_QUIET_PERIOD_INTERVAL);
 
-        scheduledTaskQueue().add(quietPeriodTask);
+        taskScheduler().schedule(quietPeriodTask);
```

Comment:
**Scalability Issue**: The modification to use taskScheduler().schedule directly for quietPeriodTask in GlobalEventExecutor could introduce a delay in the execution of this task due to the overhead of task scheduling. In a highly scalable system, even minor delays introduced in core system tasks can amplify, impacting overall system responsiveness and throughput. <br> **Fix**: Review the scheduling requirements for quietPeriodTask to determine if it can be executed immediately or if a more efficient scheduling strategy can be applied. For tasks critical to system performance, consider bypassing the task scheduler and executing directly or using a more lightweight scheduling mechanism. <br> **Code Suggestion**: 
 ```
 protected AbstractScheduledEventExecutor(TaskSchedulerFactory taskSchedulerFactory) {
     this.taskSchedulerFactory = requireNonNull(taskSchedulerFactory, "taskSchedulerFactory");
 }

 protected TaskScheduler taskScheduler() {
     if (taskScheduler == null && taskSchedulerFactory != null) {
         taskScheduler = taskSchedulerFactory.newTaskScheduler(this);
     }
     if (taskScheduler == null) {
         taskScheduler = DefaultTaskSchedulerFactory.INSTANCE.newTaskScheduler(this);
     }
     return taskScheduler;
 }

 private static boolean isNullOrEmpty(TaskScheduler taskScheduler) {
     return taskScheduler == null || taskScheduler.isEmpty();
 }

 protected final void cancelScheduledTasks() {
     assert inEventLoop();
     TaskScheduler taskScheduler = this.taskScheduler;
     if (isNullOrEmpty(taskScheduler)) {
         return;
     }
     taskScheduler.cancelScheduledTasks();
 }
 ```